content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -126,6 +119,260 @@ The Java agent logging mode. The following 3 modes are supported:
+| ---------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `OTEL_TRACES_EXPORTER=otlp`                                | Select the OpenTelemetry exporter for tracing (default).                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `OTEL_METRICS_EXPORTER=otlp`                               | Select the OpenTelemetry exporter for metrics (default).                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `OTEL_LOGS_EXPORTER=otlp`                                  | Select the OpenTelemetry exporter for logs (default).                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `OTEL_EXPORTER_OTLP_ENDPOINT`                              | The OTLP traces, metrics, and logs endpoint to connect to. Must be a URL with a scheme of either `http` or `https` based on the use of TLS. If protocol is `http/protobuf` the version and signal will be appended to the path (e.g. `v1/traces`, `v1/metrics`, or `v1/logs`). Default is `http://localhost:4317` when protocol is `grpc`, and `http://localhost:4318/v1/{signal}` when protocol is `http/protobuf`.                                          |
```

Comment:
Could be in a followup PR, but I'd love to combine these signal-specific rows into a single table row per option. As is, its hard to read as is, and could just as easy be described with `OTEL_EXPORTER_OTLP_{SIGNAL}_ENDPOINT` format, where we state the values for `{SIGNAL}` in a footnote or in the description.

Should also state somewhere that signal-specific env vars take priority over the generic version. Again, not something that needs to necessarily be done in this PR since this reflects the state of things right now.

Reply:
i made this change and added a section in the "General" section explaining the precedence